auditlog/registry.py

```diff
@@ -68,6 +72,8 @@ def register(
         mapping_fields: Optional[Dict[str, str]] = None,
         mask_fields: Optional[List[str]] = None,
         m2m_fields: Optional[Collection[str]] = None,
+        serialize_data: bool = False,
+        serialize_kwargs: Optional[dict] = None,
```

Comment:
cant we set the default value to `{}`?

```suggestion
        serialize_kwargs: Dict[str, Any] = {},
```

Reply:
No, this will store the reference to the mutable dict in the function object! Check this:
```python
def add(k, v, d={}):
    d[k] = v
    return d

print(add("key1", "value1"))  # -> {"key1": "value1"}
print(add("key2", "value2"))  # -> {"key1": "value1", "key2": "value2"}
```

Reply:
Ah yes! forgot about this. Thanks for pointing.

Reply:
How about that. Very interesting. The more you know the more you know you don't know anything.